From 4d317755a3b908cc2dada13619f08bae6d741944 Mon Sep 17 00:00:00 2001
From: serge-sans-paille <serge.guelton@telecom-bretagne.eu>
Date: Sun, 13 Dec 2020 22:12:16 +0100
Subject: [PATCH] Make RNG adaptor compatible with libstdc++

---
 pythran/pythonic/random/shuffle.hpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pythran/pythonic/random/shuffle.hpp b/pythran/pythonic/random/shuffle.hpp
index 3d3b9eb91..19b74344f 100644
--- a/pythran/pythonic/random/shuffle.hpp
+++ b/pythran/pythonic/random/shuffle.hpp
@@ -30,13 +30,13 @@ namespace random
       }
 
       typedef unsigned result_type;
-      result_type min()
+      static constexpr result_type min()
       {
         return 0;
       }
       /* -1 because of the floor() operation performed by the float->unsigned
        * conversion */
-      result_type max()
+      static constexpr result_type max()
       {
         return std::numeric_limits<result_type>::max() - 1;
       }
