From a0571440f5ba08ab3bbfb8aa01831904dfd96815 Mon Sep 17 00:00:00 2001
From: serge-sans-paille <serge.guelton@telecom-bretagne.eu>
Date: Sat, 1 Jan 2022 10:02:43 +0100
Subject: [PATCH] Numpy no longer supports floordiv on complex for python 3.10

---
 pythran/tests/test_complex.py | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/pythran/tests/test_complex.py b/pythran/tests/test_complex.py
index f835ff3760..22a60afc29 100644
--- a/pythran/tests/test_complex.py
+++ b/pythran/tests/test_complex.py
@@ -60,11 +60,6 @@ def test_complex_array_abs(self):
                       np.array([[3 + 2j]]),
                       test_complex_array_abs=[NDArray[complex, :, :]])
 
-    def test_complex_floordiv(self):
-        self.run_test('def complex_floordiv(x): import numpy as np; return np.floor_divide(x, 2 + 2j)',
-                      3.5 - 3.5j,
-                      complex_floordiv=[complex])
-
     def test_complex_array_sqr(self):
         self.run_test('def test_complex_array_sqr(a): return a ** 2',
                       np.array([[3 + 2j]]),
